case-studies/simple/draft1.md

```diff
@@ -0,0 +1,215 @@
+
+## The Problem
+
+When _Resolve to Save Lives_ approached nilenso in May of 2018, the problem they had in hand was seemingly straight-forward: Build an MVP (Minimum Viable Product) of the Simple app and start showing it to users. But even as we claim "build an MVP" to be the problem statement we start to see the different scales at which this project operates. Yes, it's true that the initial task could be summed up as "build an MVP" but from _Resolve's_ perspective, the initial-initial task was actually "find a consulting company in India to build an MVP." Halfway through 2018, there was much about the project which was uncertain, unknown, or undecided. Although "Simple" was a front-runner, the product didn't even have a name yet.
+
+Unravelling the true scale of a project is often best accomplished by asking "why?" Every paid consultant has gone through the "Five Whys" exercise. The exercise is easy: ask "why?" of the problem at hand, then ask "why?" of the answer you uncover. Repeat five times. This exercise often makes inexperienced consultants come across like curious, but incredibly annoying, three-year-olds. When consulting with an organization like _Resolve,_ the exercise wasn't actually necessary. Or perhaps it would be more accurate to say was not obviously necessary.
+
+The first meeting at the nilenso office was with Daniel Burka, Director of Product and Design at _Resolve._ The most common appraisal Daniel had among nilenso staff who hadn't met him yet was "the guy who designed the Firefox logo." For those who had started careers early enough to witness the early days of the Firefox web browser, this was no specious laurel. Firefox represented far more than the buzzwords we associate with it today: Open Source, Free Software, a World Wide Web based on Open Standards. Back in 2003, Firefox represented the democratization of the web in a David v. Goliath fight against Microsoft's stagnant and under-performing Internet Explorer, universally hated by software developers the world over. Firefox sat in the intersection of two communities, two ideas. On the left was _Free Software._ On the right was _Design_ with a Capital D. Good Design meant that a product like Firefox had to be approached with Design Thinking. It had to have a brand. It had to be fun. It had to be so much better than everything else you would be a fool not to use it. Firefox was all these things. It was also a Javascript debugger, a CSS reverse-engineering tool, and the doorway millions of developers stepped through to create the web and the internet we all rely on today. The significance of such a tool, and such a brand, is not to be understated. So, come. Meet Daniel. The guy who designed the Firefox logo.
```

Comment:
Yeah, sorry... this was mostly based on my own memory of the project kicking off. I don't remember the early conversations with (or about) Tim and all I remember of Daniel is honestly the Firefox logo stuff. I actually really like this passage, since Firefox and Simple both share some pretty commendable ideals in the intersection of design, branding, and open source licensing. These are uncommon qualities and I'd love to maintain a highlight on them somehow. This passage wasn't really about Daniel, per se. Maybe we could water down the focus on him as a character or at least remove this part entirely:

`So, come. Meet Daniel. The guy who designed the Firefox logo.`